kalitectl.py

```diff
@@ -219,7 +219,7 @@ def get_pid():
     listen_port = getattr(settings, "CHERRYPY_PORT", LISTEN_PORT)
 
     # Timeout is 1 second, we don't want the status command to be slow
-    conn = httplib.HTTPConnection(LISTEN_ADDRESS, listen_port, timeout=1)
+    conn = httplib.HTTPConnection(LISTEN_ADDRESS, listen_port, timeout=3)
```

Comment:
The first time the server starts up, on Pi's and similar servers, it can take > 15 seconds to load the first page, then it's fast(er) after that. But I think it may be possible to connect to the port well before that, so hopefully this will be sufficient.